src/Mvc/Mvc.ApiExplorer/test/EndpointMetadataApiDescriptionProviderTest.cs

```diff
@@ -50,23 +50,6 @@ public void UsesApplicationNameAsControllerNameIfNoDeclaringType()
             Assert.Equal(nameof(EndpointMetadataApiDescriptionProviderTest), apiDescription.ActionDescriptor.RouteValues["controller"]);
         }
 
-        [Fact]
-        public void AddsJsonRequestFormatWhenFromBodyInferred()
-        {
-            static void AssertJsonRequestFormat(ApiDescription apiDescription)
-            {
-                var requestFormat = Assert.Single(apiDescription.SupportedRequestFormats);
-                Assert.Equal("application/json", requestFormat.MediaType);
-                Assert.Null(requestFormat.Formatter);
-            }
-
-            AssertJsonRequestFormat(GetApiDescription(
-                (InferredJsonClass fromBody) => { }));
-
-            AssertJsonRequestFormat(GetApiDescription(
-                ([FromBody] int fromBody) => { }));
-        }
```

Comment:
I think this and the other from-body coverage we lost can be re-added with tests using TestEndpointRouteBuilder.

Reply:
Aren't those covered with the metadata tests in ```EndpointMetadataApiDescriptionProviderTest.cs``` ? What will TestEndpointRouteBuilder test ?

Reply:
I assume you mean DelegateEndpointRouteBuilderExtensionsTest? That should continue testing that the correct `IAcceptsMetadata` gets added.

We still need to test that the `IAcceptsMetadata` gets used and properly turned into the proper `SupportedRequestFormats` and `ParameterDescriptions`.

Reply:
See the following test ```MapPost_BuildsEndpointWithCorrectEndpointMetadata``` in ```DelegateEndpointRouteBuilderExtensionsTest```.  It's exactly checking that we have the correct metadata. I can add an assertion to check if the type isRequired.  Anything else ? or Can you point me to an additional test class ?

Reply:
That doesn't test anything about the `ApiDescription` that gets generated though.

Reply:
I'll admit this I'm asking for more of an integration test, but we should have those and they already exist in this test class.

Reply:
I added some more end to end tests.